tests/test_organizations.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pybuildkite.organizations import Organizations
+from pybuildkite.client import Client
+
+
+@pytest.fixture()
+def setup_org(mocker):
+    mocker.patch('pybuildkite.client.Client.get', return_value='PASS')
+    client = Client()
+    url = 'https://api.buildkite.com/v2'
+    org = Organizations(client, url)
+    yield org
+
+def test_organization(setup_org):
+    assert setup_org.client == setup_org.client
+    assert setup_org.path == 'https://api.buildkite.com/v2'+ 'organizations/'
+
+def test_list_all(setup_org):
+    result = setup_org.list_all()
```

Comment:
These arent really testing any functionality, instead you’re really just testing that your mock worked. Consider https://github.com/pyasi/pybuildkite/blob/master/tests/test_jobs.py as an example

Reply:
Hi, Thank you for example but I don't think I am only checking my mock works or not. Instead I am just mocking get function function in client class because we are just checking organisation class functions not client class alternatively you can also check coverage it's covering 100% organizations.py code. I just want a suggestion from you. Meanwhile I will do the same like the example you provided.